lms/envs/aws.py

```diff
@@ -565,13 +565,54 @@
 
 
 ##### EDX-NOTIFICATIONS ######
-NOTIFICATION_CLICK_LINK_URL_MAPS = ENV_TOKENS.get('NOTIFICATION_CLICK_LINK_URL_MAPS', NOTIFICATION_CLICK_LINK_URL_MAPS)
-NOTIFICATION_STORE_PROVIDER = ENV_TOKENS.get('NOTIFICATION_STORE_PROVIDER', NOTIFICATION_STORE_PROVIDER)
-NOTIFICATION_CHANNEL_PROVIDERS = ENV_TOKENS.get('NOTIFICATION_CHANNEL_PROVIDERS', NOTIFICATION_CHANNEL_PROVIDERS)
+NOTIFICATION_CLICK_LINK_URL_MAPS = ENV_TOKENS.get(
```

Comment:
I think I've asked this before, but it would be really nice if we could modularize these settings files to keep related subconfigurations separate.  I don't think the fork is the right place to begin doing that, however.